linux/LinuxProcessList.c

```diff
@@ -1003,14 +1003,14 @@ static int handleNetlinkMsg(struct nl_msg* nlmsg, void* linuxProcess) {
       memcpy(&stats, nla_data(nla_next(nla_data(nlattr), &rem)), sizeof(stats));
       assert(lp->super.pid == (pid_t)stats.ac_pid);
 
-      unsigned long long int timeDelta = stats.ac_etime * 1000 - lp->delay_read_time;
-      #define BOUNDS(x) (isnan(x) ? 0.0 : ((x) > 100) ? 100.0 : (x))
-      #define DELTAPERC(x,y) BOUNDS((float) ((x) - (y)) / timeDelta * 100)
-      lp->cpu_delay_percent = DELTAPERC(stats.cpu_delay_total, lp->cpu_delay_total);
-      lp->blkio_delay_percent = DELTAPERC(stats.blkio_delay_total, lp->blkio_delay_total);
-      lp->swapin_delay_percent = DELTAPERC(stats.swapin_delay_total, lp->swapin_delay_total);
-      #undef DELTAPERC
-      #undef BOUNDS
+      if (stats.ac_etime * 1000 > lp->delay_read_time) {
+         unsigned long long int timeDelta = stats.ac_etime * 1000 - lp->delay_read_time;
+         #define DELTAPERC(x, y) MINIMUM((float)saturatingSub(x, y) / timeDelta * 100.0F, 100.0F)
```

Comment:
Please keep these lowercase …
```suggestion
         #define DELTAPERC(x, y) MINIMUM((float)saturatingSub(x, y) / timeDelta * 100.0f, 100.0f)
```